File: python/project2.py
num = int(input("Please input your number: "))
sum = 0
x1 = 0
x2 = 1
x3 = x1+x2
while x3 < num:
    if x3%2==0:
	sum+=x3
    x1 = x2
    x2 = x3
    x3 = x1+x2

print"The sum is", sum

File: ruby/project_2.rb
print "Please input your number: "
number = gets.chomp.to_i
x_1 = 0
x_2 = 1
x_3 = x_1+x_2
sum = 0
x = 0
while x_3 < number
  if x_3%2 == 0
    sum = x_3+sum
    x = x+1
  end
  x_1 = x_2
  x_2 = x_3
  x_3 = x_1+x_2
end
puts "The sum of the even numbers is #{sum}."
puts "The length of all the even numbers is #{x}."
